pkg/cli/commands/rpkg/push/command.go

```diff
@@ -254,7 +254,12 @@ func readFromDir(dir string) (map[string]string, error) {
 	}); err != nil {
 		return nil, err
 	}
-	return resources, nil
+	newRes := make(map[string]string)
+	for key, value := range resources {
```

Comment:
I hate being this picky again, sorry in advance :)

Have you considered to do the transformation of the `resources` map keys where the values are added to the map (in line 252) instead of using a second for cycle.
I am thinking of something along the lines of:
`resources[filepath.ToSlash(rel)] = string(contents)`

Reply:
hi @kispaljr ,

you are absolutely right, done as you suggested. I also hate redundant loops :)
thanks for reviewing!